sql/thriftserver/src/gen/thrift/gen-javabean/org/apache/spark/sql/thriftserver/cli/thrift/TArrayTypeEntry.java

```diff
@@ -0,0 +1,363 @@
+/**
+ * Autogenerated by Thrift Compiler (0.12.0)
```

Comment:
The code generated in Hive master seems to use 0.9.3. To avoid diffs, could you compile and use 0.9.3 from https://www.apache.org/dist/thrift/0.9.3/, and add to the readme to use it?

Reply:
> The code generated in Hive master seems to use 0.9.3. To avoid diffs, could you compile and use 0.9.3 from https://www.apache.org/dist/thrift/0.9.3/, and add to the readme to use it?

Since local is 0.12..... I will use 0.93 version to rebuild this.

Reply:
@AngersZhuuuu Could we just add `hive-service-rpc` to our maven dependency?
```xml
    <dependency>
      <groupId>org.apache.hive</groupId>
      <artifactId>hive-service-rpc</artifactId>
      <version>2.3.6</version>
    </dependency>
```

Reply:
@wangyum We would be creating a dependency on a specific Hive version then again, because `ThriftCLIService.java` and `ThriftCLIServiceClient.java` and a few others depend on the exact version of the RPCs. Better to embed these.